### PR TITLE
Bound monospace formatting & other minor fixes

### PR DESCRIPTION
**Pull request by k-mack**
Read through the document and noticed a couple of things that needed to be fixed. 2daccc6 contains the only critical fix w.r.t. to asciidoctor syntax. The third commit is a nitpick that can be disregarded if it's not your style.

### CODE REVIEW

File: contents/index.adoc [hunk 1]
```diff
@@ -146,8 +146,8 @@ NOTE: The Gradle classloader automatically injects setter methods alongside all
 include::{samplescodedir}/capture-user-input/buildSrc/src/main/java/BinaryRepositoryExtension.java[]
 
 ----
-ƒ
-The task property also defines the `serverUrl` with type `Property`. It allows for mapping the state of the property without actually accessing its value until needed for processing - that is in the task action.
+
+The task property also defines the `serverUrl` with type `Property`. It allows for mapping the state of the property without actually accessing its value until needed for processing--that is in the task action.
```
Comment (bmuschko) [inline contents/index.adoc]: What's the problem with this? It seems to render just fine.

File: contents/index.adoc [hunk 2]
```diff
@@ -217,7 +217,7 @@ include::{samplescodedir}/react-to-task/buildSrc/src/main/java/InhouseConvention
 
 === Providing default dependencies for plugins
 
-The implementation of a plugin sometimes requires the use of an external dependency. You might want to automatically download an artifact using Gradle’s dependency management mechanism and later use it in the action of a task type declared in the plugin. Optimally, the plugin implementation doesn’t need to ask the user for the coordinates of that dependency - it can simply predefine a sensible default version.
+The implementation of a plugin sometimes requires the use of an external dependency. You might want to automatically download an artifact using Gradle’s dependency management mechanism and later use it in the action of a task type declared in the plugin. Optimally, the plugin implementation doesn’t need to ask the user for the coordinates of that dependency--it can simply predefine a sensible default version.
```
Comment (bmuschko) [inline contents/index.adoc]: What's the problem with this? It seems to render just fine.

File: contents/index.adoc (hunk 1 above)
Comment (k-mack) [reply contents/index.adoc]: I think I just got carried away here. All I did was convert the hypen to an em dash. I can remove that change as its related to style. The same goes for your other comment.

Comment (bmuschko) [reply contents/index.adoc]: Ah, ok. Let's remove it for now. Happy to merge afterward!

**Comment by k-mack** [comment]
@bmuschko I dropped that last commit (hypen to em dash) and rebased onto master. Should be good to go.

**Comment by bmuschko** [comment]
Thanks for your contribution. It has been merged.

**Pull request merged**